.github/workflows/build.yaml

```diff
@@ -85,10 +83,9 @@ jobs:
         runs-on: "ubuntu-latest"
 
         strategy:
+            fail-fast: false
```

Comment:
should we remove these now?

Reply:
For public repos they don't really matter. They just save resources, but I will remove them for the environment ;)